includes/external-connection-cpt.php

```diff
@@ -233,12 +235,8 @@ function ajax_verify_external_connection() {
 		exit;
 	}
 
-	$auth = array();
-	if ( ! empty( $_POST['auth'] ) ) {
-		$auth = $_POST['auth'];
-	}
-
-	$current_auth = get_post_meta( intval( sanitize_key( $_POST['endpointId'] ) ), 'dt_external_connection_auth', true );
+	$auth         = filter_input( INPUT_POST, 'auth', FILTER_REQUIRE_ARRAY );
```

Comment:
`$auth` was assigned to `false` in my test, which causes a fatal error for the `array_merge` call below.

![image](https://user-images.githubusercontent.com/5423135/142392196-0da7b50e-699c-4bbd-acf6-63ce244a6a9d.png)